### PR TITLE
delete address api !!

### PR DESCRIPTION
**Pull request by swaadheenta**

### CODE REVIEW

File: lib/view/Screens/AddressScreen/components/Addresslist.dart [hunk 1]
```diff
@@ -24,6 +24,7 @@ class AddressList extends StatelessWidget {
                     padding: const EdgeInsets.only(bottom: 16.0),
                     child: AddressBox(
                         isSelected: index == controller.selectedAddressIndex,
```
Comment (Alpha17-2) [inline lib/view/Screens/AddressScreen/components/Addresslist.dart:26]: remove the isSelected argument from AddressBox

**Review by Alpha17-2 — CHANGES_REQUESTED** [review]
Changes required

File: lib/view/Screens/AddressScreen/components/Addresslist.dart (hunk 1 above)
Comment (swaadheenta) [reply lib/view/Screens/AddressScreen/components/Addresslist.dart]: Requested changes has been made !!

**Review by Alpha17-2 — APPROVED** [review]
Well done @swaadheenta

**Review by Alpha17-2 — APPROVED** [review]

**Pull request merged**